gui/Makefile

```
@@ -0,0 +1,14 @@
+CXX = g++
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)